Quantum-Control-Applications/Superconducting/Two-Flux-Tunable-Transmons/Use Case 2 - Two-Qubit-Randomized-Benchmarking/two_qubit_rb_example.py

```diff
@@ -36,16 +35,21 @@ def multiplexed_readout(I, I_st, Q, Q_st, resonators, sequential=False, amplitud
 ##  Two-qubit RB functions  ##
 ##############################
 # assign a string to a variable to be able to call them in the functions
-q1 = "1"
-q2 = "2"
+q1_idx_str = "1"
+q2_idx_str = "2"
 
 
-# single qubit generic gate constructor Z^{z}Z^{a}X^{x}Z^{-a} that can reach any point on the Bloch sphere (starting from arbitrary points)
+# single qubit generic gate constructor Z^{z}Z^{a}X^{x}Z^{-a}
+# that can reach any point on the Bloch sphere (starting from arbitrary points)
 def bake_phased_xz(baker: Baking, q, x, z, a):
-    element = f"q{q}_xy"
-    baker.frame_rotation_2pi(-a, element)
+    if q == 1:
+        element = f"q{q1_idx_str}_xy"
+    else:
+        element = f"q{q2_idx_str}_xy"
```

Comment:
Why not just passing the qubit number here as before?

Reply:
This is for when elements are described by some other labelling rule. E.g. on the chip, they are labelled qubit 20 and qubit 45, and this is reflected by the QUA elements. That is why you need to map q1 and q2 values 1 and 2 onto some label `q1_idx_str` and `q2_idx_str`